### ISSUE TITLE
Organize README.md files in repo

### ISSUE THREAD

**Issue by yaaminiv** [hputnam/Meth_Compare#91]
- Write up README files so there is organization for all other files
- Separate folders for main figures and supplemental figures
- Separate folders for main code and exploratory code
- Once framework is in place make sure all files are in the appropriate folder

**Comment by yaaminiv** [comment]
Went through and tried adding more structure to all the README.md files. Here's what's left that I may need help with:

- [main README.md](https://github.com/hputnam/Meth_Compare/blob/master/README.md): Add links to output (tables/figures) and code used to generate output
- [code README.md](https://github.com/hputnam/Meth_Compare/blob/master/code/README.md): Add numbers to the beginning of the name of each script so it's easier to determine an order for running everything
- [genome feature tracks README.md](https://github.com/hputnam/Meth_Compare/blob/master/genome-feature-files/Readme.md): Make sure external links for all genome information are correct

Also created separate folders for supplementary information within analyses, code, and output. Will post a new issue to help organize files in these subdirectories.

**Comment by yaaminiv** [comment]
better reflected in this issue: https://github.com/hputnam/Meth_Compare/issues/95
